CRM/Civicase/Service/CaseCategoryFromUrl.php

```diff
@@ -99,9 +98,17 @@ private function getCaseCategoryFromUrl($caseTypeCategoryParam) {
     $caseCategory = CRM_Utils_Request::retrieve($caseTypeCategoryParam, 'String');
     if ($caseCategory) {
       if (is_numeric($caseCategory)) {
-        $caseTypeCategories = CaseType::buildOptions($caseTypeCategoryParam, 'validate');
```

Comment:
Any reason why we changed the implementation here?

Reply:
I did this change when the parameter name was also going to change (to ctc), but now it has no sense, and it was complicating the code. I reverted it: https://github.com/compucorp/uk.co.compucorp.civicase/pull/756/commits/7341da3cd4725ffb0bfd64db67c6ac82d1d1eb5a